### ISSUE TITLE
Unable to start from within parent folder with name containing 'src'

### ISSUE THREAD

**Issue by FrancoisCamus** [sfeir-open-source/sfeir-school-pwa#48]
I attended this training and was unable to npm start it on my computer and had to borrow another one. The issue came from the fact that I was running it from a parent folder named GiHub-src and there is an issue in doing so with cross-var: https://github.com/elijahmanor/cross-var/issues/17

Changing the name of the folder fixes the issue :)

**Comment by jefBinomed** [comment]
I look at the issue and I don't understand which directory I have to change?

**Comment by FrancoisCamus** [comment]
It is a bug in cross-var. As a student if I clone your repo is a folder with a name containing 'src' then npm start fails.
So in my case I cloned it in a folder named GiHub-src that's why it was failing.

**Comment by jefBinomed** [comment]
Ok, so I have noting to do :'(

**Comment by FrancoisCamus** [comment]
Yeah I just raised it so you know about it as it may happen to your students in the future.
